src/Terrain/terrain.cpp

```diff
@@ -595,15 +593,15 @@ const OnePath Terrain::get_path_type(int xs, int ys, int zs, int xf, int yf, int
     int dz = 3;
     int dxy = 1;
 
-    int8_t type = abs(xs - xf) + abs(ys - yf) + 4 * abs(zs - zf);
+    OnePath type(((abs(xs - xf) + abs(ys - yf)) << (1 + 2 * abs(zs - zf))) + (abs(zs - zf) << 1));
```

Comment:
Ok you need to comment this.

And not put it in the initializer

Reply:
lol